typings/index.d.ts

```diff
@@ -2573,7 +2662,7 @@ export interface WebhookFields extends PartialWebhookFields {
   readonly createdTimestamp: number;
   delete(reason?: string): Promise<void>;
   edit(options: WebhookEditData, reason?: string): Promise<Webhook>;
-  sendSlackMessage(body: object): Promise<boolean>;
+  sendSlackMessage(body: unknown): Promise<boolean>;
```

Comment:
Is it not possible to type this?

Reply:
We'd need to type it off of Slack's documentation..

Reply:
Yeah let's not.